galactus.theme

```diff
@@ -201,8 +201,22 @@ function galactus_preprocess_html(&$variables) {
   }
 
   // Add a class to define the active CLF theme colour
-  $clfcolour = theme_get_setting('clf_theme_colour');
-  $variables['attributes']['class'][] = 'clf-theme-' . $clfcolour;
```

Comment:
I worry if someone has added custom styles to this that maybe this causes issues, could we keep it or will it break things?

Reply:
That's a good point. Yes, best to leave this in. I'll add it back